cypress/integration/Fields/PDQCitations/index.js

```diff
@@ -0,0 +1,52 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+//This step is just a workaround until it gets fixed
+Given('user is navigating to {string}', (a) => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message like 'list not defined' etc.
+        // and don't want to fail the test so we return false
+        if (err.message.includes(`Cannot read property 'getAttribute' of undefined`)) {
+            return false
+        }
+    });
+    cy.visit(a);
+});
+
+Then('pdq citation section is visible', () => {
+    cy.get('div.pdq-sections > ol').should('be.visible');
+});
+
+And('pdq citation header text is {string}', (headerText) => {
+    cy.get('div.pdq-sections > h6').should('be.visible').first().and('have.text', headerText);
+});
+
+And('pdq citation number {string} is a PubMed link with an url {string}', (pubMedLinkNumber, linkUrl) => {
+    const linkPath = linkUrl.split(',');
+    const number = pubMedLinkNumber.split(',');
+    for (let i = 0; i < linkPath.length; i++) {
+        cy.get(`li[id*='section']`).eq(number[i]).find('a').should('have.attr', 'href', linkPath[i]);
+    }
+});
+
+And('total number of pub med citations is {int}', (totalPubMed) => {
+    cy.get('div.pdq-sections > ol').first().find(`a[title*='PubMed']`).and('have.length', totalPubMed);
+});
+
+And('total number of pdq citations is {int}', (totalPdqCitation) => {
+    cy.get('div.pdq-sections > ol').first().find('li').and('have.length', totalPdqCitation);
+});
```

Comment:
same
And Pdq citation section number 1 has a total <number> of citations

Reply:
Done...Changed the step to say:
"And pdq citation section number 1 has a total number of citations of <totalPdqCitation>"

Reply:
same - make use of passed number of section pls